tests/ansible-target-files/write_env_to_file.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+env > ./conjur_env.txt
```

Comment:
Is this this why the tests branch needs to be merged into master?  I don't see any new tests, so I'm going to assume that the ones in the molecule branch will work seamlessly with these.  Is that a safe assumption?

Reply:
This is just for us to see that the module works. I use this file as the command to run in the module (`/conjurinc/ansible/write_env_to_file`) because in the `command` module you can't write `>`.

The tests in the molecule branch work fine with this